README.md

```diff
@@ -44,7 +44,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
 
 
-- [coming soon]
+- (https://derrick1451.github.io/Mobile-Portfolio/)
```

Comment:
- [OPTIONAL]  Great work adding the project deployment link to the `README`. 👏🏾 I want to suggest a different approach to how to add links to a `README`. You can do it like this [Live demo](https://derrick1451.github.io/Mobile-Portfolio/) ` [Live demo](https://derrick1451.github.io/Mobile-Portfolio/)`. This improves the link appearance in the `README` making it look professional. You can do this by using the chain link icon in the `README` file. To learn more about [markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax), please use the link provide.